src/components/form/form.js

```diff
@@ -519,6 +402,8 @@ class Form extends React.Component {
 
     if (valid && this.props.onSubmit) {
       this.props.onSubmit(ev, valid, this.enableForm);
+    } else if (valid) {
+      this._form.submit();
     }
```

Comment:
this conditional block could be extracted into a separate method like `triggerFormSubmit`

Reply:
could the `this.props.afterFormValidation` conditional be moved above the `valid` conditional and then reuse that `valid` conditional to submit the form?

Reply:
done 👍